content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+      "https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly",\
+      "https://www.googleapis.com/auth/trace.append" \
+      --num-nodes "4" --network "default" --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --async
+    {{< /text >}}
+
+1.  Wait for clusters to transition to `RUNNING` state:
+
+    {{< text bash >}}
+    $ gcloud container clusters list
+    {{< /text >}}
+
+1.  Get the clusters' credentials ([command details](https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials)):
+
+    {{< text bash >}}
+    $ gcloud container clusters get-credentials cluster-1 --zone $zone
+    $ gcloud container clusters get-credentials cluster-2 --zone $zone
+    {{< /text >}}
+
+1.  Validate `kubectl` access to each cluster:
+
+    1.  Check cluster-1
+
+        {{< text bash >}}
+        $ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Check cluster-2:
+
+        {{< text bash >}}
+        $ kubectl config use-context "gke_${proj}_${zone}_cluster-2"
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+1.  Create a `cluster-admin` cluster role binding tied to the kubernetes credentials associated with your GCP user.
+    _Note:_ replace `mygcp@gmail.com` with the email tied to your Google Cloud account:
+
+    {{< text bash >}}
+    $ KUBE_USER="mygcp@gmail.com"
+    $ kubectl create clusterrolebinding gke-cluster-admin-binding \
+      --clusterrole=cluster-admin \
+      --user="${KUBE_USER}"
+    {{< /text >}}
+
+## Create a Google Cloud firewall rule
+
+The following rule needs to be created to allow each clusters' pods to directly communicate:
+
+{{< text bash >}}
+$ function join_by { local IFS="$1"; shift; echo "$*"; }
+$ ALL_CLUSTER_CIDRS=$(gcloud container clusters list --format='value(clusterIpv4Cidr)' | sort | uniq)
+$ ALL_CLUSTER_CIDRS=$(join_by , ${ALL_CLUSTER_CIDRS})
+$ ALL_CLUSTER_NETTAGS=$(gcloud compute instances list --format='value(tags.items.[0])' | sort | uniq)
+$ ALL_CLUSTER_NETTAGS=$(join_by , ${ALL_CLUSTER_NETTAGS})
+$ gcloud compute firewall-rules create istio-multicluster-test-pods \
+  --allow=tcp,udp,icmp,esp,ah,sctp \
+  --direction=INGRESS \
+  --priority=900 \
+  --source-ranges="${ALL_CLUSTER_CIDRS}" \
+  --target-tags="${ALL_CLUSTER_NETTAGS}" --quiet
+{{< /text >}}
+
+## Install the Istio control plane
+
+The following generates an Istio installation manifest, installs it, and enables automatic sidecar injection in
+the `default` namespace:
+
+{{< text bash >}}
+$ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio_master.yaml
+$ kubectl create ns istio-system
+$ kubectl apply -f $HOME/istio_master.yaml
+$ kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+Wait for pods to come up:
```

Comment:
This text is misleading, the command does not wait for the pods to come up. Please consider rewriting this text to be clear about what the reader needs to do and what the command actually does.